src/main/resources/templates/profile_details_page.html

```diff
@@ -75,6 +75,9 @@ <h1 class="text-center">Profile Details</h1>
                 <label class="form-label">Profile Picture</label>
                 <input class="form-control" type="file" @change="handleFileUpload" :disabled="state.uploadingImage">
                 <p class="form-text">Image must be below 1024x1024px. Use PNG or JPG format.</p>
```

Comment:
There is also file format validation needed. I suggest removing support of PNG and do only JPG.

Reply:
Yep, on way to go.